src/qt/locale/bitcoin_es.ts

```diff
@@ -1148,7 +1176,7 @@ El proceso de migración creará un respaldo del monedero antes de migrar. Este
     </message>
     <message>
         <source>Are you sure you wish to close the wallet &lt;i&gt;%1&lt;/i&gt;?</source>
```

Comment:
Another translation with HTML in it.

Reply:
This comes from:https://github.com/bitcoin/bitcoin/blob/f490f5562d4b20857ef8d042c050763795fd43da/src/qt/walletcontroller.cpp#L94